code/workspaces/web-app/src/components/notebooks/NotebookSharingOptions.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  select: {
+    whiteSpace: 'normal',
+    maxWidth: 500,
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+  },
+  title: {
+    fontWeight: 500,
+  },
+  description: {
+    marginTop: theme.spacing(1),
+  },
+  warning: {
+    color: 'hsl(0, 40%, 40%)',
+  },
```

Comment:
Very minor but, as above, the change in font size has altered things slightly. It's not necessary, but I think it looks a bit more balanced when this class `theme.spacing(0.5)`